Version 3 + Vaadin 24 fails to export (errors on Preview)

Seems to be unable to find some xdev and jasper packages? Any advice?
```

2023-06-07 10:59:56,723 ERROR [com.vaadin.flow.server.DefaultErrorHandler] (default task-2) : java.lang.RuntimeException: Failed to export
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.jasper.format.AbstractJasperReportFormat.export(AbstractJasperReportFormat.java:135)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.jasper.format.AbstractJasperReportFormat.exportToBytes(AbstractJasperReportFormat.java:109)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.jasper.format.AbstractJasperReportFormat.export(AbstractJasperReportFormat.java:86)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.wizard.steps.PreviewStep.onEnterStep(PreviewStep.java:61)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.wizard.steps.PreviewStep.onEnterStep(PreviewStep.java:33)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.components.wizard.panel.WizardPanel.showStep(WizardPanel.java:200)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.components.wizard.panel.WizardPanel.showNextStep(WizardPanel.java:179)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.components.wizard.buttonbar.AbstractWizardButtonBar.lambda$addButtonClickEvent$7c90d456$1(AbstractWizardButtonBar.java:95)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.component.ComponentEventBus.fireEventForListener(ComponentEventBus.java:233)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.component.ComponentEventBus.handleDomEvent(ComponentEventBus.java:475)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.component.ComponentEventBus.lambda$addDomTrigger$dd1b7957$1(ComponentEventBus.java:292)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.internal.nodefeature.ElementListenerMap.lambda$fireEvent$2(ElementListenerMap.java:447)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.internal.nodefeature.ElementListenerMap.fireEvent(ElementListenerMap.java:447)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.rpc.EventRpcHandler.handleNode(EventRpcHandler.java:62)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.rpc.AbstractRpcInvocationHandler.handle(AbstractRpcInvocationHandler.java:74)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.ServerRpcHandler.handleInvocationData(ServerRpcHandler.java:438)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.ServerRpcHandler.lambda$handleInvocations$1(ServerRpcHandler.java:419)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:419)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:320)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:114)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.VaadinService.handleRequest(VaadinService.java:1529)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.server.VaadinServlet.service(VaadinServlet.java:399)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//com.vaadin.flow.spring.SpringServlet.service(SpringServlet.java:106)
	at jakarta.servlet.api@6.0.0//jakarta.servlet.http.HttpServlet.service(HttpServlet.java:614)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at org.wildfly.security.elytron-web.undertow-server@3.0.1.Final//org.wildfly.elytron.web.undertow.server.ElytronRunAsHandler.lambda$handleRequest$1(ElytronRunAsHandler.java:68)
	at org.wildfly.security.elytron-base@2.1.0.Final//org.wildfly.security.auth.server.FlexibleIdentityAssociation.runAsFunctionEx(FlexibleIdentityAssociation.java:103)
	at org.wildfly.security.elytron-base@2.1.0.Final//org.wildfly.security.auth.server.Scoped.runAsFunctionEx(Scoped.java:161)
	at org.wildfly.security.elytron-base@2.1.0.Final//org.wildfly.security.auth.server.Scoped.runAs(Scoped.java:73)
	at org.wildfly.security.elytron-web.undertow-server@3.0.1.Final//org.wildfly.elytron.web.undertow.server.ElytronRunAsHandler.handleRequest(ElytronRunAsHandler.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.RedirectDirHandler.handleRequest(RedirectDirHandler.java:68)
	at io.undertow.core@2.3.6.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.core@2.3.6.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:258)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler.dispatchToServlet(ServletInitialHandler.java:193)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.spec.RequestDispatcherImpl.forwardImpl(RequestDispatcherImpl.java:186)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.spec.RequestDispatcherImpl.forwardImplSetup(RequestDispatcherImpl.java:136)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.spec.RequestDispatcherImpl.forward(RequestDispatcherImpl.java:99)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.mvc.ServletForwardingController.handleRequestInternal(ServletForwardingController.java:141)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.mvc.AbstractController.handleRequest(AbstractController.java:178)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter.handle(SimpleControllerHandlerAdapter.java:51)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1081)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:974)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1011)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:914)
	at jakarta.servlet.api@6.0.0//jakarta.servlet.http.HttpServlet.service(HttpServlet.java:547)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:885)
	at jakarta.servlet.api@6.0.0//jakarta.servlet.http.HttpServlet.service(HttpServlet.java:614)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:129)
	at io.undertow.websocket@2.3.6.Final//io.undertow.websockets.jsr.JsrWebSocketFilter.doFilter(JsrWebSocketFilter.java:172)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:231)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:365)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:100)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:179)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:227)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:221)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:107)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:93)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.csrf.CsrfFilter.doFilterInternal(CsrfFilter.java:117)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:82)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:62)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:233)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:191)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:352)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:268)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.boot.web.servlet.support.ErrorPageFilter.doFilter(ErrorPageFilter.java:124)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.boot.web.servlet.support.ErrorPageFilter$1.doFilterInternal(ErrorPageFilter.java:99)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.boot.web.servlet.support.ErrorPageFilter.doFilter(ErrorPageFilter.java:117)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at org.wildfly.security.elytron-web.undertow-server@3.0.1.Final//org.wildfly.elytron.web.undertow.server.ElytronRunAsHandler.lambda$handleRequest$1(ElytronRunAsHandler.java:68)
	at org.wildfly.security.elytron-base@2.1.0.Final//org.wildfly.security.auth.server.FlexibleIdentityAssociation.runAsFunctionEx(FlexibleIdentityAssociation.java:103)
	at org.wildfly.security.elytron-base@2.1.0.Final//org.wildfly.security.auth.server.Scoped.runAsFunctionEx(Scoped.java:161)
	at org.wildfly.security.elytron-base@2.1.0.Final//org.wildfly.security.auth.server.Scoped.runAs(Scoped.java:73)
	at org.wildfly.security.elytron-web.undertow-server@3.0.1.Final//org.wildfly.elytron.web.undertow.server.ElytronRunAsHandler.handleRequest(ElytronRunAsHandler.java:67)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.RedirectDirHandler.handleRequest(RedirectDirHandler.java:68)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:117)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.core@2.3.6.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.core@2.3.6.Final//io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.core@2.3.6.Final//io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at org.wildfly.security.elytron-web.undertow-server-servlet@3.0.1.Final//org.wildfly.elytron.web.undertow.server.servlet.CleanUpHandler.handleRequest(CleanUpHandler.java:38)
	at io.undertow.core@2.3.6.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.security.jacc.JACCContextIdHandler.handleRequest(JACCContextIdHandler.java:61)
	at io.undertow.core@2.3.6.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.deployment.GlobalRequestControllerHandler.handleRequest(GlobalRequestControllerHandler.java:68)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.SendErrorPageHandler.handleRequest(SendErrorPageHandler.java:52)
	at io.undertow.core@2.3.6.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:276)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:132)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1430)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1430)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1430)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1430)
	at org.wildfly.extension.undertow@28.0.1.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1430)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:256)
	at io.undertow.servlet@2.3.6.Final//io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:101)
	at io.undertow.core@2.3.6.Final//io.undertow.server.Connectors.executeRootHandler(Connectors.java:393)
	at io.undertow.core@2.3.6.Final//io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:859)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1990)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)
	at org.jboss.xnio@3.8.9.Final//org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1282)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: software.xdev.dynamicreports.report.exception.DRException: net.sf.jasperreports.engine.JRException: Errors were encountered when compiling report expressions class file:
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:18: error: cannot find symbol
public class Report_1686160795700_909252 extends JREvaluator
                                                 ^
  symbol: class JREvaluator
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:25: error: cannot find symbol
    private JRFillParameter parameter_CUSTOM_VALUES = null;
            ^
  symbol:   class JRFillParameter
  location: class Report_1686160795700_909252
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:4: error: package net.sf.jasperreports.engine does not exist
import net.sf.jasperreports.engine.*;
^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:5: error: package net.sf.jasperreports.engine.fill does not exist
import net.sf.jasperreports.engine.fill.*;
^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:48: error: cannot find symbol
        parameter_CUSTOM_VALUES = (JRFillParameter)pm.get("CUSTOM_VALUES");
                                   ^
  symbol:   class JRFillParameter
  location: class Report_1686160795700_909252
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:79: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_10_"); //$JR_EXPR_ID=8$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:84: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_12_"); //$JR_EXPR_ID=9$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:107: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_10_"); //$JR_EXPR_ID=8$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:112: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_12_"); //$JR_EXPR_ID=9$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:135: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_10_"); //$JR_EXPR_ID=8$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:140: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_12_"); //$JR_EXPR_ID=9$
                                                                  ^
11 errors
.
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.dynamicreports.jasper.builder.JasperReportBuilder.toJasperReport(JasperReportBuilder.java:308)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.dynamicreports.jasper.builder.JasperReportBuilder.toJasperPrint(JasperReportBuilder.java:349)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.dynamicreports.jasper.builder.JasperReportBuilder.export(JasperReportBuilder.java:710)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.dynamicreports.jasper.builder.JasperReportBuilder.toText(JasperReportBuilder.java:644)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.vaadin.grid_exporter.jasper.format.AbstractJasperReportFormat.export(AbstractJasperReportFormat.java:131)
	... 160 more
Caused by: net.sf.jasperreports.engine.JRException: Errors were encountered when compiling report expressions class file:
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:18: error: cannot find symbol
public class Report_1686160795700_909252 extends JREvaluator
                                                 ^
  symbol: class JREvaluator
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:25: error: cannot find symbol
    private JRFillParameter parameter_CUSTOM_VALUES = null;
            ^
  symbol:   class JRFillParameter
  location: class Report_1686160795700_909252
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:4: error: package net.sf.jasperreports.engine does not exist
import net.sf.jasperreports.engine.*;
^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:5: error: package net.sf.jasperreports.engine.fill does not exist
import net.sf.jasperreports.engine.fill.*;
^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:48: error: cannot find symbol
        parameter_CUSTOM_VALUES = (JRFillParameter)pm.get("CUSTOM_VALUES");
                                   ^
  symbol:   class JRFillParameter
  location: class Report_1686160795700_909252
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:79: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_10_"); //$JR_EXPR_ID=8$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:84: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_12_"); //$JR_EXPR_ID=9$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:107: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_10_"); //$JR_EXPR_ID=8$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:112: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_12_"); //$JR_EXPR_ID=9$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:135: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_10_"); //$JR_EXPR_ID=8$
                                                                  ^
C:\wildfly-28.0.1.Final-DEV\bin\Report_1686160795700_909252.java:140: error: package software.xdev.dynamicreports.jasper.base does not exist
                value = ((software.xdev.dynamicreports.jasper.base.JasperCustomValues)parameter_CUSTOM_VALUES.getValue()).getValue("simpleExpression_12_"); //$JR_EXPR_ID=9$
                                                                  ^
11 errors
.
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//net.sf.jasperreports.engine.design.JRAbstractCompiler.compileReport(JRAbstractCompiler.java:233)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//net.sf.jasperreports.engine.JasperCompileManager.compile(JasperCompileManager.java:358)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//net.sf.jasperreports.engine.JasperCompileManager.compileReport(JasperCompileManager.java:618)
	at deployment.mwsjconfig-1.0-SNAPSHOT.war//software.xdev.dynamicreports.jasper.builder.JasperReportBuilder.toJasperReport(JasperReportBuilder.java:306)
	... 164 more
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 3.0.1
+* Fixed compilation problems due to missing ``ecj`` dependency #98
+* Updated dependencies
+
 ## 3.0.0
 ⚠️<i>This release contains breaking changes</i>
 

--- a/vaadin-grid-exporter/pom.xml
+++ b/vaadin-grid-exporter/pom.xml
@@ -108,7 +108,7 @@
 		<dependency>
 			<groupId>com.xdev-software</groupId>
 			<artifactId>dynamicreports-core-for-grid-exporter</artifactId>
-			<version>1.0.2</version>
+			<version>1.0.3</version>
 		</dependency>
 
 		<dependency>

